File: pertemuan3/PR/PR03_2473021_PY/B_MaxMin.py
"""
Pertemuan 03
[PR] Membuat Program dimana user diminta untuk menginput 3 nilai bilangan bulat yaitu a,b,c dan mengkategorikan berdasarkan angka terbesar, tengah, dan terkecil

@author 2473021-Febrianus Leona Putra
@version 06 Oktober 2024

"""
# Input
print("Input :")
a = int(input("a : "))
b = int(input("b : "))
c = int(input("c : "))

# Process
if (a > b and a > c):
    terbesar = a
elif (b > a and b > c):
    terbesar = b
else:
    terbesar = c

if (a < b and a < c):
    terkecil = a
elif (b < a and b < c):
    terkecil = b
else:
    terkecil = c

if (a != terbesar and a != terkecil):
    tengah = a
elif (b != terbesar and b != terkecil):
    tengah = b
else:
    tengah = c

# Output
print("Output :")
print("Nilai",tengah,"di antara",terbesar,"dan",terkecil)
print("Nilai terbesar adalah",terbesar)
print("Nilai terkecil",terkecil)

File: pertemuan3/PR/PR03_2473021_JAVA/B_MaxMin.java
/**
 * Pertemuan 03
 * [PR] Membuat Program dimana user diminta untuk menginput 3 nilai bilangan bulat yaitu a,b,c dan mengkategorikan berdasarkan angka terbesar, tengah, dan terkecil
 *
 * @author 2473021-Febrianus Leona Putra
 * @version 6 Oktober 2024
 */
package pertemuan3.PR.PR03_2473021_JAVA;

import java.util.Scanner;

public class B_MaxMin {

    public static void main(String[] args) {
        // Input
        Scanner sc = new Scanner(System.in);
        System.out.println("Input :");
        System.out.print("a : ");
        int a = sc.nextInt();
        System.out.print("b : ");
        int b = sc.nextInt();
        System.out.print("c : ");
        int c = sc.nextInt();

        // Process
        int terbesar;
        if (a > b && a > c) {
            terbesar = a;
        } else if (b > a && b > c) {
            terbesar = b;
        } else {
            terbesar = c;
        }

        int terkecil;
        if (a < b && a < c) {
            terkecil = a;
        } else if (b < a && b < c) {
            terkecil = b;
        } else {
            terkecil = c;
        }

        int tengah;
        if (a != terbesar && a != terkecil) {
            tengah = a;
        } else if (b != terbesar && b != terkecil) {
            tengah = b;
        } else {
            tengah = c;
        }

        // Output
        System.out.println("Output :");
        System.out.println("Nilai " + tengah + " di antara " + terbesar + " dan " + terkecil);
        System.out.println("Nilai terbesar adalah " + terbesar);
        System.out.println("Nilai terkecil " + terkecil);

        sc.close();
    }
}
